src/dynarec/rv64/dynarec_rv64_helper.h

```diff
@@ -233,6 +247,15 @@
     j64 = (dyn->insts)?(dyn->insts[ninst].epilog-(dyn->native_size)):0;\
     B(j64)
 
+// Branch to MARKSEG if reg is 0 (use j64)
+#define CBZ_MARKSEG(reg)    \
+    j64 = GETMARKSEG-(dyn->native_size);   \
+    BEQZ_safe(reg, j64);
```

Comment:
no need to use the safe version here, the jump will be withing reach all the time for MARK* conditionnal jump

Reply:
Fixed, but I don't really understand how `maskseg` works, just copied from ARM port, can you explain a little?